deployment/monocular/values.yaml

```diff
@@ -34,11 +34,16 @@ api:
       #- name: my-repo-name
       #  url: my-repository-url
       #  source: my-repository-source
+    cors:
+      allowed_origins:
+        - my-api-server
```

Comment:
This is the default in https://github.com/helm/monocular/blob/master/src/api/config/cors/cors.go#L30, I just copied it here. It means that only a hostname of `my-api-server` or same-origin will work. A blank value would allow all origins, I believe.